src/main/java/thePackmaster/cards/monsterhunterpack/AwakenedRitualDagger.java

```diff
@@ -18,7 +18,7 @@ public class AwakenedRitualDagger extends AbstractMonsterHunterCard {
     public static final int UPG_DAMAGE = 10;
 
     public AwakenedRitualDagger() {
-        super(ID, 1, CardType.ATTACK, CardRarity.SPECIAL, CardTarget.ENEMY);
+        super(ID, 1, CardType.ATTACK, CardRarity.SPECIAL, CardTarget.ENEMY, CardColor.COLORLESS);
```

Comment:
For all these Monster Hunter cards, is there are consideration to leaving them as the Packmaster color? They feel different to me than all the other tokens, since they get added to your deck permanently instead of being generated in combat. 

I don't really have a strong opinion either way, but while making everything generated during combat colorless makes sense, these are a bit different. They're almost like transformations of the Carving Knife that was used to kill the enemy and gain the new card.

Reply:
I'd prefer them not being colorless for the reason you stated. I feel like they're an integral part of the pack and not really a token that you generate often